common/analysis/chainController/create.js

```diff
@@ -0,0 +1,16 @@
+import * as ChainFactory from '../chainFactory'
+import * as ChainUpdate from './chainUpdate'
+import { assocCalculation } from './assocCalculation'
+
+export const createStep = ({ chain, props = {} }) => {
+  const step = ChainFactory.createStep({ chain, props })
+  return { ...ChainUpdate.assocStep({ chain, step }), step }
+}
+
+export const createCalculation = ({ chain, step, nodeDefUuid = null, props = {} }) => {
```

Comment:
method names should not match the factory ones. call this createAndAssocCalculation

Reply:
ok, right

Reply:
also, please rename the file according to the exported function

Reply:
ok, done